src/utils/export_mapping.py

```diff
@@ -0,0 +1,124 @@
+"""
+Copyright 2020 Google LLC.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    https://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+
+import ee
+
+
+# Function uses Earth Engine API
+# Contains more local variables than rocommended in a Python code
+def images_with_stacked_bands(multispectral_image, wind, dsm, road, tropomi,
+                              vertical_kernel, horizontal_kernel, num_samples,
+                              bands, scale):
+    """Stack bands from different imageries.
+
+    Parameters
+    ----------
+    multispectral_image : ee.Image
+        Multispectral Image
+    wind : CollectionClass
+        collectionClass for Wind Imagery
+    dsm : CollectionClass
+        collection class for DSM Imagery
+    road : RoadImagery
+        Road Imagery
+    tropomi : CollectionClass
+        Tropomi imagery
+    vertical_kernel : ee.Kernel
+        kernel used for convolved mask to calculate valid pixels
+    horizontal_kernel : ee.Kernel
+        horizontal kernel for convolution
+    num_samples : int
+        Minimum number of valid pixels that stacked image shoudl have
+    bands : list[str]
+        list of bands that the image shoudl contain
+    scale : int
+        Scale for clipping and sampling
+
+    Returns
+    -------
+    ee.ImageCollection
+        Collection of images that contain all requested bands with num_samples
+        valid pixels.
+
+    """
+    start_time = multispectral_image.get('collectionStartTime')
+    multispectral_date = ee.Date(start_time)
```

Comment:
Since we don't use start_time anywhere else, there is no reason to have a separate variable for start_time.
You can just do:   
    multispectral_date = ee.Date(multispectral_image.get('collectionStartTime'))

Reply:
Just to avoid a long line of code